app/assets/javascripts/components/bulkUploadImport.jsx

```diff
@@ -327,7 +364,7 @@ class BulkUploadImport extends React.Component {
                     <label htmlFor={i}> {sample.name}</label>
                   </p>
                   <div className="col s4">
-                    <select className="" id={i} onChange={ this.handleProjectChangeForSample } value={sample.project_id}>
+                    <select className="" id={i} onChange={ this.handleProjectChangeForSample } value={this.state.project}>
```

Comment:
why do you set the value to this.state.project? individual samples can map to different project id.

Reply:
Just noticed that all sample project ids are being updated on project change, so no need to change then individually, i would revert it, thanks.